src/demos/graph/1-basic-graph/src/app.js

```diff
@@ -1,17 +1,68 @@
 /* global window */
 import React, {Component} from 'react';
 
+// data
+import sampleGraph from '../../data/sample-graph';
+
+// utils
+import Graph from '../../common/graph';
+
+// components
+import GraphRender from './graph-render'
+
+function randomPosition(width, height) {
+  const halfWidth = width / 2;
+  const halfHeight = height / 2;
+  const x = Math.random() * width - halfWidth;
+  const y = Math.random() * height - halfHeight;
+  return [x, y];
+}
+
 export default class App extends Component {
 
   constructor(props) {
     super(props);
+    this.state = {
+      viewport: {
+        width: window.innerWidth,
+        height: window.innerHeight
+      },
+      graph: new Graph()
+    };
+  }
+
+  componentDidMount() {
+    this.processData();
+  }
+
+  processData() {
+    if (sampleGraph) {
```

Comment:
i'd still remove it ;) will do that too in the building-a-geospatial app

Reply:
will do